src/reports/package/reporter-factory.ts

```diff
@@ -40,8 +41,11 @@ const axeResultsReportGenerator = (parameters: AccessibilityInsightsReport.AxeRe
     const reactStaticRenderer = new ReactStaticRenderer();
     const fixInstructionProcessor = new FixInstructionProcessor();
 
+    const sectionFactory = { ...AutomatedChecksReportSectionFactory,
+        FooterSection: PackageReportFooter};
```

Comment:
I like the idea of overriding section at the root composition level but this leads to duplication on the new footer component. Basically, `ReportFooter` (under `src\reports\components\report-sections\report-footer.tsx`) and the new `PackageReportFooter` are almost exactly the same. I put some more comments on `PackageReportFooter` directly

Reply:
Refactored to remove duplication. Some textual duplication remains but there should be virtually no code duplication.